src/ServiceControl.SagaAudit/SagaHistoryCleaner.cs

```diff
@@ -77,33 +77,21 @@ public static void Clean(int deletionBatchSize, DocumentDatabase database, DateT
 
             var deletionCount = Chunker.ExecuteInChunks(items.Count, (itemsForBatch, db, s, e) =>
             {
-                if (logger.IsDebugEnabled)
-                {
-                    logger.Debug($"Batching deletion of {s}-{e} saga history documents.");
-                }
+                logger.Debug($"Batching deletion of {s}-{e} saga history documents.");
```

Comment:
This is a more general comment for all changes in this PR, where we still use interpolation. For consistency, would it make sense to also use the format-based overloads, like you did in some other changes in this PR?

Reply:
I've actually gone the other way and have gone for interpolation rather than using the format overloads. My reasons:

* It's easier to read what the value when compared to looking at the index number and comparing to the args array
* If we in the future support structured logging, we may be able to pull more meaning from the logs.

Reply:
I'm not an expert on this. If I read this right, we would have to use the template-based approach outlined in 
https://docs.microsoft.com/en-us/aspnet/core/fundamentals/logging/?view=aspnetcore-6.0#log-message-template
docs.microsoft.comdocs.microsoft.com

Which means "not using interpolation".